contents/handbook/growth/marketing/newsletter.md

```diff
@@ -4,24 +4,75 @@ sidebar: Handbook
 showTitle: true
 ---
 
-We regularly send two emails. 
+We regularly send three emails. 
 
-1. What's New In PostHog?, a product announcement email sent every month. Sent via Customer.io.
-2. HogMail, a marketing and content email published every two weeks.
+1. Changelog, a product announcement email sent every month. Sent via Customer.io. (Owner: Joe)
```

Comment:
I think you can remove all the owners here as you mention them below in each section anyway